res/database/Default/user/kieferer.xml

```diff
@@ -120,20 +133,25 @@ But the tour operator tricks them.</en>
 			<title>
 				<de>Goldrausch%MODIFIER%</de>
 				<en>Gold Rush%MODIFIER%</en>
+				<pl>Gorączka złota%MODIFIER%</pl>
 			</title>
 			<description>
 				<de>Rivalisierende Goldschürfer wetteifern in %RANDOMCITY% um die größten Goldfunde.
 Dabei sind ihnen alle Mittel recht.</de>
 				<en>Rival gold diggers compete for the biggest gold finds in %RANDOMCITY%.
 Any means will do.</en>
+				<pl>Rywalizujący poszukiwacze złota rywalizują o największe znaleziska złota w %RANDOMCITY%.
+Każdy środek się nada.</pl>
 			</description>
 			<variables>
 				<modifier>
 					<de>| in %RANDOMCITY%</de>
 					<en>| in %RANDOMCITY%</en>
+					<pl>| w %RANDOMCITY%</pl>
 				</modifier>
 				<randomcity>
 					<de>%STATIONMAP:RANDOMCITY%|Prosperous Creek</de>
+					<pl>Prosperous Creek|Sakramento|Ontario|Zimnym potoku|Bloody Village</pl>
```

Comment:
number of alternatives must be the same

Reply:
It will not be grammatically consistent. Changed

Reply:
I adapted it grammatically